app/pods/cluster/template.hbs

```diff
@@ -71,18 +71,21 @@
     {{/dashboard-module}}
 
     {{#dashboard-module label='Nodes'}}
-        {{#if model.riakNodes}}
+        {{#if model.nodes}}
             <ul class="button-list">
-                {{#each model.riakNodes as |node|}}
+                {{#each model.nodes as |node|}}
                     <li>
-                        <a href="node-stats?nodeId={{node.id}}&clusterId={{model.id}}"
-                           class={{unless node.available "danger"}}>
-                            {{node.id}}
-                            <span
-                                class="glyphicon {{if node.available "glyphicon-ok-circle" "glyphicon-ban-circle"}}"
-                                aria-hidden="true">
-                      </span>
-                        </a>
+                        {{#if node.available}}
+                            {{#link-to 'node' model.id node.id }}
+                                <span class="glyphicon glyphicon-ok-circle" aria-hidden="true"></span>
+                                {{node.id}}
+                            {{/link-to}}
+                        {{else}}
+                            {{#link-to 'node' model.id node.id class='danger'}}
+                                <span class="glyphicon glyphicon-ban-circle" aria-hidden="true"></span>
+                                {{node.id}}
+                            {{/link-to}}
+                        {{/if}}
```

Comment:
The previous code was cleaner, but because I was not able to use the `link-to` helper because of that wierd bug I mentioned a while back and was instead using the `<a>` tag, it was causing a full page refresh. This is more verbose, but its works as intended.